package.json

```diff
@@ -69,18 +69,19 @@
     "@babel/preset-env": "7.18.2",
     "@babel/preset-react": "7.17.12",
     "@babel/register": "7.17.7",
-    "@rollup/plugin-babel": "5.3.0",
-    "@rollup/plugin-commonjs": "22.0.0",
+    "@rollup/plugin-babel": "5.3.1",
+    "@rollup/plugin-commonjs": "24.0.1",
     "@rollup/plugin-node-resolve": "13.3.0",
+    "@types/estree": "0.0.50",
```

Comment:
CircleCI failed because of some very weird bug, which I suspect is related to some package using an very old version of this dependency, so I needed to bypass that by installing a newer one on the top-level:

![image](https://user-images.githubusercontent.com/845031/214592975-c8a3a3bc-9522-470f-b756-929ec3fd8257.png)

Apparently this  started happening to many people:
https://stackoverflow.com/q/67806963/104380